elide-graphql/src/main/java/com/yahoo/elide/graphql/GraphQLConversionUtils.java

```diff
@@ -44,39 +43,15 @@
     protected NonEntityDictionary nonEntityDictionary = new NonEntityDictionary();
     protected EntityDictionary entityDictionary;
 
+    private final Map<Class, GraphQLObjectType> outputConversions = new HashMap<>();
+    private final Map<Class, GraphQLInputObjectType> inputConversions = new HashMap<>();
+    private final Map<Class, GraphQLEnumType> enumConversions = new HashMap<>();
```

Comment:
Do we need a separate map for enumConversions?  Can we just use outputConversions?

Reply:
`GraphQLEnumType`s are both `GraphQLInputObjectType` and `GraphQLOutputObjectType`. As a result, we can share this across inputs and outputs. In fact, we may have to so we avoid naming collisions.